example-playbooks/manage_users/manage_users.yml

```diff
@@ -0,0 +1,91 @@
+---
+
+#     This playbook allows the creation of cql users and deletes the default cassandra user as recommended in scylla's
+#     documentation -> {https://docs.scylladb.com/stable/operating-scylla/security/enable-authorization.html#enabling-authorization}.
+#     By default this playbook will add the users scylla_admin (superuser), scylla_cql_monitor and customer_admin,
+#     but you can change the list of users and permissions by overriding the variable users.
+#     Every user defined in this playbook must be also defined in the inventory file under the section [cql_credentials] followed
+#     by its password.
+#     E.g.:
+#     [cql_credentials]
+#     scylla_admin=password1
+#     scylla_cql_monitor=password2
+#     customer_admin=password3
+#
+#     Operations allowed:
+#       * Addition of users
+#       * Addition of permissions for any user (Removing permissions isn't supported)
+#       * Changing password for any user except scylla_admin
+#
+#     Usage:
+#       ansible-playbook -i inventory.ini manage_users.yaml [-e "@users.yml"]
+
+- name: Add users from inventory and delete cassandra user
+  hosts: scylla
+  vars_files:
+    - vars/main.yml
+  tasks:
+    - name: Get cql address and check if authentication is enabled
+      block:
+        - command: cat /etc/scylla/scylla.yaml
+          ignore_errors: true
```

Comment:
How come? How can you continue if it errors out? Wouldn't it make the following `"{{ _scylla_yaml_out.stdout | from_yaml }}"` fail too?

Reply:
Fixed